app/actions/action_updatePins.js

```diff
@@ -1,18 +1,45 @@
-import { UPDATE_PINS } from '../constants/constants.js';
+import { UPDATE_PINS, SET_TARGET } from '../constants/constants.js';
 import { userData } from '../lib/db/db.js';
-function updatePins(payload) {
+import { Alert } from 'react-native';
+const updatePins = (payload) => {
```

Comment:
How come this action creator uses a function expression (with `const`) while the others are all declarations?